src/message/messageActionSheet.js

```diff
@@ -16,18 +20,60 @@ const copyToClipboard = async ({ message, auth }) => {
   Clipboard.setString(rawMessage);
 };
 
-export const actionSheetButtons = [
-  'Narrow to conversation',
-  'Reply',
-  'Copy to clipboard',
-  'Cancel'
+const unMuteTopicHelper = ({ auth, message }) => {
+  unMuteTopic(auth, message.display_recipient, message.subject);
+};
+
+const muteTopicHelper = ({ auth, message }) => {
+  muteTopic(auth, message.display_recipient, message.subject);
+};
+
+const unMuteStreamHelper = ({ auth, message, subscriptions }) => {
+  const sub = subscriptions.find(x => x.name === message.display_recipient);
+  unMuteStream(auth, sub.stream_id);
+};
+
+const muteStreamHelper = ({ auth, message, subscriptions }) => {
+  const sub = subscriptions.find(x => x.name === message.display_recipient);
+  muteStream(auth, sub.stream_id);
+};
+
+const skip = () => false;
+
+const actionSheetButtons = [
```

Comment:
We'll need to translate all these strings.
Ping me when you finish the other things to decide how to do it.